lockapi.pm

```diff
@@ -141,6 +141,10 @@ sub barrier_wait (@args) {
             return 1;
         }
 
+        if (looks_like_number($timeout) && time > $start + $timeout) {
+            bmwqemu::mydie "barrier '$name' timeout after " . (time - $start) . "s";
+        }
+
```

Comment:
```suggestion
        bmwqemu::mydie "barrier '$name' timed out after " . (time - $start) . "s" if looks_like_number($timeout) && time > $start + $timeout;
```

No need for all those additional blank lines or other lines :)

Reply:
If you think this is more readable, okay.

What about `timeout => -1`  maybe we should treat it (or all negative values) as infinitive as well?

Reply:
I changed the timeout behavior that only positive numbers including 0 are valid. Everything else will be treated as infinitive.

Also, do not wait longer then timeout, if it doesn't match the poll interval.